fragments/filter/README.md

```diff
@@ -0,0 +1,786 @@
+to provide an out-of-band way for users to discover what operators from the Simple CQL class are implemented. After OAFeat CQL is
+finalized, implementations will be expected to provide correct conformance classes with respect to their implementation of CQL.
+
+An additional change that may be made to the Simple CQL conformance class is to only require support of expressions with a property
+name of the left hand side and a literal on the right hand side (e.g., `eo:cloud_cover <= 10`), and additional conformance classes to
+will support arbitrary uses of properties and literals in expressions on either side. The primary motivation for this is to allow 
+implementations that use datastores that do not easily support right-hand side properties to implement Simple CQL 
+(e.g., Elasticsearch). Implementers should feel free to only implement `property operand literal` expressions at this time.
+
+## Getting Started with Implementation
+
+It recommended that implementers start with fully implementing only a subset of functionality. As stated previously, 
+until the OAFeat CQL spec is finalized, it is legal in a STAC API implementation to advertise that the Simple CQL conformance 
+class is implemented, but that only a subset of that functionality is implemented. 
+
+A good place to start is 
+implementing only the logical and simple comparison operators (`=`, `<`, `<=`, `>`, `>=`), defining a static Queryables 
+schema with no queryables advertised, and only implementing CQL Text. Following from that can be support for 
+INTERSECTS and ANYINTERACTS, defining a static Queryables schema with only the basic Item properties, and also 
+implementing CQL JSON. From there, other comparison operators can be implemented and a more 
+dynamic Queryables schema (if desired).
+
+Formal definitions and grammars for CQL can be found here: 
+
+- The [OAFeat (CQL) spec](https://portal.ogc.org/files/96288) includes an ABNF for cql-text and both JSON Schema and 
+  OpenAPI specifications for cql-json. The standalone files are:
+  - [cql.bnf](https://github.com/opengeospatial/ogcapi-features/blob/master/extensions/cql/standard/schema/cql.bnf)
+  - [cql.json](https://github.com/opengeospatial/ogcapi-features/blob/master/extensions/cql/standard/schema/cql.json)
+  - [cql.yml](https://github.com/opengeospatial/ogcapi-features/blob/master/extensions/cql/standard/schema/cql.yml)
+- A JSON Schema for only the parts of the CQL JSON encoding required by this extension is [here](cql.json)
+- A OpenAPI specification for only the parts of the CQL JSON encoding required by this extension is [here](cql.yml)
+- xtraplatform-spatial has a CQL [ANTLR 4 grammer](https://github.com/interactive-instruments/xtraplatform-spatial/tree/master/xtraplatform-cql/src/main/antlr/de/ii/xtraplatform/cql/infra)
+
+These projects have or are developing CQL support:
+
+- [GeoPython PyCQL](https://github.com/geopython/pycql/tree/master/pycql), and the 
```

Comment:
Should https://github.com/geopython/pygeofilter be used instead, as it seams to be more aligned with the current OAFeatures Filtering spec? This is a bit of a guess, as I haven't grokked both libraries completely, but this seems to be under more active development

Reply:
Thanks for the pointer to that. I added in pygeofilter, but I also don't know what either provides. Looks like they do have a cql-json impl, whereas I think pyCQL is only doing cql-text right now.